src/Storages/MergeTree/StorageFromMergeTreeDataPart.h

```diff
@@ -45,6 +45,16 @@ class StorageFromMergeTreeDataPart final : public ext::shared_ptr_helper<Storage
         return part->storage.getVirtuals();
     }
 
+    String getPartitionId() const
+    {
+        return part->info.partition_id;
+    }
+
+    String getPartitionIDFromQuery(const ASTPtr & ast, const Context & context) const
+    {
+        return part->storage.getPartitionIDFromQuery(ast, context);
+    }
+
```

Comment:
These two methods would become unneeded if you removed `MutationCommand::isPartitionAffected(...)` method

Reply:
That's not right, they are not being used in this method at all.